python/boat_controller_node.py

```diff
@@ -24,8 +24,18 @@
 # Global variables for configuring controller (should be command line arguments ideally)
 DISABLE_LOW_POWER = False
 
-controller = HeadingController(boat_speed=0, disableLowPower=DISABLE_LOW_POWER)
-
+"""
+No Fixed State: None
+Jibe: 1
+Tack: 2
+Low Power: 3
+"""
+FIXED_CTRL_STATE = None
+
+# Boat control logic
+controller = HeadingController(boat_speed=0, disableLowPower=DISABLE_LOW_POWER, fixedControlMode=FIXED_CTRL_STATE)
```

Comment:
@patrick-5546 I was originally thinking of adding command line arguments for this instead. However, when I do this it seems to cause errors and it will also interfere with Navigation since they are not aware of command line arguments from us.

Do you know know a better way other than hardcoding these constants in?